10-angular/custom_directives/readme.md

```diff
@@ -493,18 +502,18 @@ To start, make a second directive in your Javascript just by copying and pasting
 
 ```js
 (function(){
-  var directives = angular.module('grumbleDirectives',[]);
-  directives.directive('grumbleShow', function(){
+  angular
+  .module("grumbles")
+  .directive('grumbleShow', function(){
     return {
-      templateUrl: "views/grumbles/_grumble_show.html",
+      templateUrl: "js/grumbles/_grumble_show.html",
       replace: true,
       scope: {
         grumble: "="
       }
     }
-  });
-
-  directives.directive('grumbleSave', function(){
+  })
+  .directive('grumbleSave', function(){
```

Comment:
recommend calling this `grumbleForm` and putting it in a separate file called `grumble_form.directive.js`.